### PR TITLE
Update docs generation to point to the correct urls

### PR DESCRIPTION
**Pull request by manuelpuyol**
The `source` and `storybook` urls weren't taking the status module into account, so I'm updating `docs.rake` to take them into consideration

### CODE REVIEW

File: docs/src/@primer/gatsby-theme-doctocat/components/head.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ function Head(props) {
 
   let primerViewComponentsSrc
 
-  if(process.env.NODE_ENV == 'development') {
+  if(process.env.NODE_ENV === 'development') {
```
Comment (srt32) [inline docs/src/@primer/gatsby-theme-doctocat/components/head.js:21]: :)

File: lib/tasks/docs.rake [hunk 2]
```diff
@@ -385,4 +384,55 @@ namespace :docs do
 
     pretty_value(constant_value)
   end
+
+  def status_module_and_short_name(component)
+    name_with_status = component.name.gsub(/Primer::|Component/, "")
+
+    m = name_with_status.match(/(?<status>Beta|Alpha)?(::)?(?<name>.*)/)
```
Comment (srt32) [inline lib/tasks/docs.rake:391]: We don't need deprecated here?

**Review by srt32 — APPROVED** [review]
Much nicer ;)

File: lib/tasks/docs.rake (hunk 2 above)
Comment (manuelpuyol) [reply lib/tasks/docs.rake:391]: we do 😬

**Pull request merged**